.strict-typing

```diff
@@ -223,6 +223,7 @@ homeassistant.components.number.*
 homeassistant.components.nut.*
 homeassistant.components.oncue.*
 homeassistant.components.onewire.*
+homeassistant.components.onkyo.*
```

Comment:
It's very confusing that strict-typing is implemented at the same time as a config-flow PR.
Can that not be a separate/preliminary PR?